src/estimation.jl

```diff
@@ -151,81 +150,72 @@ complex, the algorithm is [^Quinn2009].
 [^Quinn2009]: B Quinn, "Recent advances in rapid frequency estimation", Digital
     Signal Processing, Vol. 19 (2009), Elsevier.
 """
-quinn(x ; kwargs...) = quinn(x, jacobsen(x, 1.0), 1.0 ; kwargs...)
+quinn(x; kwargs...) = quinn(x, jacobsen(x, 1.0), 1.0; kwargs...)
 
-quinn(x, Fs ; kwargs...) = quinn(x, jacobsen(x, Fs), Fs ; kwargs...)
+quinn(x, Fs; kwargs...) = quinn(x, jacobsen(x, Fs), Fs; kwargs...)
 
-function quinn(x::Vector{<:Real}, f0::Real, Fs::Real ; tol = 1e-6, maxiters = 20)
-    fₙ = Fs/2
-    N = length(x)
+function quinn(v::Vector{<:Real}, f0::Real, Fs::Real; tol=1e-6, maxiters::Integer=20)
+    fₙ = Fs / 2
 
     # Run a quick estimate of largest sinusoid in x
-    ω̂ = π*f0/fₙ
+    ω̂ = π * f0 / fₙ
 
     # remove DC
-    x .= x .- mean(x)
+    x = v .- mean(v)
```

Comment:
Could have left the name of the parameter `x` and just make this like `x = x .- mean(x)` to the same effect with slightly less code churn.